src/ui/backend/BaseBacking.js

```diff
@@ -16,38 +16,59 @@
 
 import setProperty from 'util/setProperty';
 
-// keys map to properties
 var BASE_STYLE_PROPS = {
-  'x': { value: 0 },
-  'y': { value: 0 },
-  // translate
-  'offsetX': { value: 0 },
-  'offsetY': { value: 0 },
-  // rotation and scale
-  'anchorX': { value: 0 },
-  'anchorY': { value: 0 },
-  'centerAnchor': { value: false },
-  'width': { cb: '_onResize' },
-  'height': { cb: '_onResize' },
-  'r': { value: 0 },
-  'opacity': { value: 1 },
-  'zIndex': {
-    value: 0,
-    cb: '_onZIndex'
-  },
-  'scale': { value: 1 },
-  'scaleX': { value: 1 },
-  'scaleY': { value: 1 },
-  'flipX': { value: false },
-  'flipY': { value: false },
-  'visible': { value: true },
-  'clip': { value: false },
-  'backgroundColor': { value: '' },
-  'compositeOperation': { value: '' }
+  x: 0,
+  y: 0,
+  offsetX: 0,
+  offsetY: 0,
+  anchorX: 0,
+  anchorY: 0,
+  centerAnchor: false,
+  _width: 0,
```

Comment:
Long term, we will probably bring native up-to-speed with the es6 branch, but it's not a priority now, and should not prevent us from making the web version as good as it can be.